src/hotspot/os_cpu/windows_x86/os_windows_x86.cpp

```diff
@@ -437,6 +437,15 @@ void os::print_context(outputStream *st, const void *context) {
   st->cr();
   st->print(  "RIP=" INTPTR_FORMAT, uc->Rip);
   st->print(", EFLAGS=" INTPTR_FORMAT, uc->EFlags);
+  // Add XMM registers + MXCSR. Note that C2 uses XMM to spill GPR values including pointers.
+  st->cr();
+  st->cr();
+  for (int i = 0; i < 16; ++i) {
+    const uint64_t *xmm = ((const uint64_t*)&(uc->Xmm0)) + 2 * i;
```

Comment:
Using a more specific cast is generally better, in this case since it's a pointer to pointer, reinterpret\_cast might suffice. And if that is used, the const is not required (I think)

Reply:
Actually, scratch that. The const is probably needed even with reinterpret\_cast. I got it mixed up with something else